connect-location/src/main/java/com/ifttt/location/ConnectionRefresher.java

```diff
@@ -49,7 +49,8 @@ public Result doWork() {
             ConnectLocation.init(getApplicationContext());
         }
 
-        ConnectionApiClient connectionApiClient = ConnectLocation.getInstance().connectionApiClient;
+        ConnectLocation connectLocation = ConnectLocation.getInstance();
```

Comment:
since this is a Singleton, do we need to assign it to a variable?

Reply:
The purpose of this is just so that we can use this instance again in the logs below, but it looks like we are not using it at the moment, will update.

Reply:
https://github.com/IFTTT/ConnectSDK-Android/pull/225/commits/e278f76aea348cf2dd57573995494fa43c39d818